src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,218 @@
+package duke;
+
+import duke.ImportExportFileHandler;
+import duke.exceptions.*;
+import duke.tasks.*;
```

Comment:
Even though all files are required, it is still advised to not do wildcard import. It's possible to configure IntelliJ to stop converting automatically.